src/modules/commander/commander.cpp

```diff
@@ -2915,18 +2915,19 @@ int commander_thread_main(int argc, char *argv[])
 			arming_state_changed = false;
 		}
 
-		/* now set navigation state according to failsafe and main state */
-		bool nav_state_changed = set_nav_state(&status,
-						       &internal_state,
-						       &mavlink_log_pub,
-						       (datalink_loss_enabled > 0),
-						       _mission_result.finished,
-						       _mission_result.stay_in_failsafe,
-						       &status_flags,
-						       land_detector.landed,
-						       (rc_loss_enabled > 0),
-						       offboard_loss_act,
-						       offboard_loss_rc_act);
+        /* now set navigation state according to failsafe and main state */
+        bool nav_state_changed = set_nav_state(&status,
```

Comment:
Everything has to be tabs and default is 8 width (if you use Sublime, you get a decent default). Anything having mixed formatting is the result of style checking not being enforced yet (e.g. in the commander). We're cleaning up module-by-module and probably should enforce style when this PR is in.